roles/openshift_node/tasks/main.yml

```diff
@@ -70,25 +70,15 @@
     - openshift_disable_swap | default(true) | bool
 # End Disable Swap Block
 
-# We have to add tuned-profiles in the same transaction otherwise we run into depsolving
-# problems because the rpms don't pin the version properly. This was fixed in 3.1 packaging.
 - name: Install Node package
   package:
-    name: "{{ openshift.common.service_type }}-node{{ openshift_pkg_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }},tuned-profiles-{{ openshift.common.service_type }}-node{{ openshift_pkg_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }}"
+    name: "{{ openshift.common.service_type }}-node{{ openshift_pkg_version | default('') | oo_image_tag_to_rpm_version(include_dash=True) }}"
```

Comment:
The actual profiles were included in a separate PR: https://github.com/openshift/openshift-ansible/pull/5054, PTAL.  I agree that the rpm dependency should get removed from the origin repo (origin.spec file), but this is something I wanted to do after the fact (PRs 5054 and 5089 are merged).  The correct profiles (new profile hierarchy) will be used even if you leave the old tuned rpm packages as a dependency.  Tested both on RHEL and RHEL Atomic Host 7.4.